analytics/management/commands/populate_realms_db.py

```diff
@@ -0,0 +1,201 @@
+from zerver.lib.server_initialization import create_internal_realm, create_users
+from zerver.lib.storage import static_path
+from zerver.lib.stream_color import STREAM_ASSIGNMENT_COLORS
+from zerver.lib.types import ProfileFieldData
+from zerver.lib.users import add_service
+from zerver.lib.utils import generate_api_key
+from zerver.models import (
+    AlertWord,
+    Client,
+    CustomProfileField,
+    DefaultStream,
+    Draft,
+    Huddle,
+    Message,
+    Reaction,
+    Realm,
+    RealmAuditLog,
+    RealmDomain,
+    RealmUserDefault,
+    Recipient,
+    Service,
+    Stream,
+    Subscription,
+    UserGroup,
+    UserMessage,
+    UserPresence,
+    UserProfile,
+    flush_alert_word,
+    get_client,
+    get_or_create_huddle,
+    get_realm,
+    get_stream,
+    get_user,
+    get_user_by_delivery_email,
+    get_user_profile_by_id,
+)
+
+
+class Command(BaseCommand):
+    help = "Populate database with different types of realms that can exist."
+
+    @override
+    def handle(self, *args: Any, **options: Any) -> None:
+        # Create a realm for each plan type
+        for plan_type, plan_name in Realm.ALL_PLAN_TYPES:
+            customer_profiles = [
+                {
+                    "unique_id": f"{plan_name}-sponsorship-pending",
+                    "sponsorship_pending": True,
+                },
+                {
+                    "unique_id": f"{plan_name}-annual-standard",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.ANNUAL,
+                    "tier": CustomerPlan.STANDARD,
+                },
+                {
+                    "unique_id": f"{plan_name}-annual-plus",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.ANNUAL,
+                    "tier": CustomerPlan.PLUS,
+                },
+                {
+                    "unique_id": f"{plan_name}-annual-enterprise",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.ANNUAL,
+                    "tier": CustomerPlan.ENTERPRISE,
+                },
+                {
+                    "unique_id": f"{plan_name}-monthly-standard",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.MONTHLY,
+                    "tier": CustomerPlan.STANDARD,
+                },
+                {
+                    "unique_id": f"{plan_name}-monthly-plus",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.MONTHLY,
+                    "tier": CustomerPlan.PLUS,
+                },
+                {
+                    "unique_id": f"{plan_name}-monthly-enterprise",
+                    "sponsorship_pending": False,
+                    "billing_schedule": CustomerPlan.ANNUAL,
+                    "tier": CustomerPlan.ENTERPRISE,
+                },
```

Comment:
Nit: This code would probably read more nicely with a simple dataclass than with dictionaries, in that you'd be able to avoid a bunch of ["foo"]` quotation marks.

Reply:
Yeah, done.